zktest/src/archive/test2/B85-ZK-3972-3.zul

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?import org.zkoss.zk.ui.util.*?>
+<!--
+B85-ZK-3972.zul
```

Comment:
It doesn't belong to this issue (Again! 👎 )

Reply:
sorry ok